rosa_tutorials/rosa-mobb-deploying-rosa-sts-mode.adoc

```diff
@@ -0,0 +1,96 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-deploying-rosa-sts-mode"]
+= Tutorial: Deploying ROSA in STS mode
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-deploying-rosa-sts-mode
+
+toc::[]
+
+// Mobb content metadata
+// Brought into ROSA product docs 2023-09-21
+// ---
+// date: '2022-05-27'
+// title: Deploying ROSA in STS mode
+// tags: ["AWS", "ROSA", "STS"]
+// weight: 1
+// aliases: ["/docs/quickstart-rosa.md"]
+//authors:
+//  - Paul Czarkowski
+//  - Michael Ducy
+// ---
+
+
+* https://youtu.be/R1T0yk9l6Ys[Quick Introduction by Ryan Niksch (AWS) and Shaozen Ding (Red Hat)].
+
+
+Secure Token Service allows you to deploy ROSA without needing a ROSA admin account. STS uses roles and policies with Amazon STS (secure token service) to gain access to the AWS resources needed to install and operate the cluster.
+
+== Prerequisites
+* xref:../rosa_tutorials/rosa-mobb-prerequisites-tutorial.adoc#rosa-mobb-prerequisites-tutorial[If this is your first time deploying ROSA, follow these prerequisite instructions].
+
+== Deploying a ROSA cluster
+
+. Set environment variables.
++
+[source,terminal]
+----
+   $ export ROSA_CLUSTER_NAME=mycluster
+    export AWS_ACCOUNT_ID=`aws sts get-caller-identity \
+      --query Account --output text`
+    export REGION=us-east-2
+    export AWS_PAGER=""
+----
+. Make sure your ROSA CLI version is correct (v1.2.25 or above).
++
+[source,terminal]
+----
+   $ rosa version
+----
+. Run the ROSA CLI to create your cluster.
+
++
+[source,terminal]
+----
+   $ rosa create cluster --sts --cluster-name ${ROSA_CLUSTER_NAME} \
+      --region ${REGION} --mode auto --yes
+----
+
+. Watch the install logs.
++
+[source,terminal]
+----
+   $ rosa logs install -c $ROSA_CLUSTER_NAME --watch --tail 10
+----
+
+== Validating the cluster
+
+Once the cluster has finished installing, validate that you can access it.
+
+. Create an Admin user.
++
+[source,terminal]
+----
+    $ rosa create admin -c $ROSA_CLUSTER_NAME
+----
+
+. Wait a few moments and run the `oc login` command it provides. If it fails, or if you get a warning about TLS certificates, wait a few minutes and try again.
+
+. Run `oc whoami --show-console`, browse to the provided URL and log in using the credentials provided above.
+
+== Cleaning up
+
+. Delete the ROSA cluster.
++
+[source,terminal]
+----
+   $ rosa delete cluster -c $ROSA_CLUSTER_NAME
+----
+. Clean up the STS roles.
++
+Once the cluster has been deleted, you can delete the STS roles. Replace <id> with the IDs from the output of the previous step.
++
+[source,terminal]
+----
+   $ rosa delete operator-roles -c <id> --yes --mode auto
+    rosa delete oidc-provider -c <id>  --yes --mode auto
+----
```

Comment:
There are two flows to create sts cluster, one uses the classic oidc configuration and create the operator-roles for specific cluster, while another flow uses the BYO oidc configuration and the operator-roles prior to cluster spec. 
In this PR, it only introduces the classic flow. Could you confirm if the BYO oidc config+operator-roles prior cluster spec flow needs to be added here. By the way the clean-up steps are using different commands.